src/project-css/app/Jobs/CreateSearchIndex.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Jobs;
+
+use App\Adapters\SearchIndexAdapter;
+use Illuminate\Bus\Queueable;
+use Illuminate\Http\Request;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Log;
+
+class CreateSearchIndex implements ShouldQueue
+{
+    use InteractsWithQueue, Queueable, SerializesModels;
+
+    public $tblNme;  
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct($tblNme)
+    {
+        $this->tblNme = $tblNme;
+    }
+
+    /**
+     * Execute the job.
```

Comment:
Needs Proper Documentation: 
Use full PHP Documentation Standards.
Be clear on what the parameters of the function are as well as what the return values are supposed to be.

You should talk about what the handle function does and how the search index adapter plays a role in this.